data/json/emit.json

```diff
@@ -117,5 +117,21 @@
     "//": "Large hot air emitter",
     "field": "fd_hot_air2",
     "qty": 100
+  },
+  {
+    "id": "emit_heater_vehicle",
+    "type": "emit",
+    "//": "Emission from vehicle heater",
+    "field": "fd_hot_air3",
+    "qty": 3,
+    "chance": 100
+  },
+  {
+    "id": "emit_cooler_vehicle",
+    "type": "emit",
+    "//": "Emission from vehicle cooler",
```

Comment:
The "cooler_vehicle" part from the id string is a dead giveaway that this has something to do with the vehicle cooler. But it's nice to see my suspicion being confirmed explicitly.